packages/terra-divider/src/terra-dev-site/doc/divider/Divider.1.doc.mdx

```diff
@@ -0,0 +1,51 @@
+import { Badge } from 'terra-divider/package.json?dev-site-package';
+
+import DividerExample from '../example/DividerExample';
+import DividerWithText from '../example/DividerWithText';
+import DividerWithWrappingText from '../example/DividerWithWrappingText';
+
+import DividerPropsTable from 'terra-divider/src/Divider?dev-site-props-table';
+
+<Badge />
+
+# Terra Divider
+
+The divider component is used to visually separate content on a layout.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-divider`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import Divider from 'terra-divider';
+```
+
+## Component Features
+
+* [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+* [LTR/RTL Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#ltr--rtl-support)
+
+[View component source code](https://github.com/cerner/terra-core/tree/master/packages/terra-divider)
+
+## Examples
+<DividerExample />
+<DividerWithText />
+<DividerWithWrappingText />
```

Comment:
These examples are missing the example template title

Reply:
updated here https://github.com/cerner/terra-core/pull/2823/commits/e331d53d05bab48ea6fe0b265d86c1c0a5aecc7c.